crux-core/src/crux/api/Crux.java

```diff
@@ -85,9 +85,9 @@ public static ICruxAPI startNode(URL url) throws IndexVersionOutOfSyncException
      * Starts a Crux node using the provided configuration.
      * <p>
      * <pre>
-     * ICruxAPI cruxNode = Crux.startNode(n -> {
+     * ICruxAPI cruxNode = Crux.startNode(buildNode(n -> {
```

Comment:
This looks like it's on the Javadoc for the overload expecting a Builder, so a separate call to `buildNode` shouldn't be necessary?

Reply:
Yep. I forgot to change the Javadoc back after adding the `Consumer` signature back in